### ISSUE TITLE
Scores for LayoutLMv2 not matching with paper result

### ISSUE THREAD

**Issue by NehaMotlani** [NielsRogge/Transformers-Tutorials#16]
Hello,

Thank you for your notebooks, they are really helpful to get started. 
I am trying to reproduce the results of the LayoutLMv2 research paper using LayoutLMv2-base and they are 2% less than the reported ones using your notebook. In the paper, they have reported precision: 0.8029, recall: 0.8539, f1-score: 0.8276 but maximum scores that I am able to get are precision: 0.7907, recall: 0.8248, f1': 0.8074. What changes can I make to your notebook to get accuracy closer to the ones mentioned in the paper?

**Comment by NielsRogge** [comment]
Hi,

I also experienced this, results can slightly differ between different runs. This is logical, as neural networks are not deterministic (unless using the same seed). However, running the notebook several times, I got the following results:

'overall_precision': 0.798961937716263, 'overall_recall': 0.8350813743218807, 'overall_f1': 0.8166224580017684

Which is in line (more or less) in line with the paper. The differences can be explained by the use of a different OCR engine (I'm using PyTesseract, whereas the original model used [Azure Cognitive Services](https://docs.microsoft.com/en-us/azure/cognitive-services/computer-vision/overview-ocr). You could also try with the OCR results already provided by the FUNSD dataset itself.

**Comment by NielsRogge** [comment]
I've uploaded a new [notebook](https://github.com/NielsRogge/Transformers-Tutorials/blob/master/LayoutLMv2/FUNSD/Fine_tuning_LayoutLMv2ForTokenClassification_on_FUNSD_using_HuggingFace_Trainer.ipynb), which uses the HuggingFace Trainer and uses the same hyperparameters as the original implementation. Now I get:

`'test_overall_precision': 0.8190854870775348, 'test_overall_recall': 0.8268941294530858, 'test_overall_f1': 0.8229712858926342'`

**Comment by NehaMotlani** [comment]
Thanks a lot. I will try out the new notebook once.
